#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("range_sensor_clip",	bool_t,		0, "Clip range sensor values to max range", True)
gen.add("range_sensor_max",	double_t,	0, "Range sensor max range", 0.5, 0.4, 6)

exit(gen.generate("wild_thumper", "wild_thumper", "WildThumper"))
